pkg/generator/templates.go

```diff
@@ -562,7 +588,12 @@ fi
 : ${IMAGE:?"Need to set IMAGE, e.g. gcr.io/<repo>/<your>-operator"}
 
 echo "building container ${IMAGE}..."
-docker build -t "${IMAGE}" -f tmp/build/Dockerfile .
+docker build -t "${IMAGE}" -f tmp/build/Dockerfile . --build-arg NAMESPACEDMAN=$NAMESPACEDMAN
```

Comment:
We're no longer using this `docker-build.sh` script anymore right? We're just calling `docker build` directly in `operator-sdk build`.
So we should probably not generate this anymore.